src/GoTrueClient.ts

```diff
@@ -478,6 +491,7 @@ export default class GoTrueClient {
   /**
    * Overrides the JWT on the current client. The JWT will then be sent in all subsequent network requests.
    * @param access_token a jwt access token
+   * @deprecated you can use `getSession()` instead as it reads from storage every time (always fresh)
```

Comment:
hmm just wondering, why are we deprecating `setAuth()` here? wouldn't this be useful for a user using gotrue-js on it's own and they need to set a custom JWT?

Reply:
Good point. Fixed in #300

We should also consider actually persisting the sessions in `setAuth()`, although it will mean we'll have to make it async.